core/block/editor/table/table.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	"github.com/anyproto/anytype-heart/core/block/editor/smartblock"
 	"github.com/anyproto/anytype-heart/core/block/editor/state"
+	"github.com/anyproto/anytype-heart/core/block/editor/table/tableutils"
```

Comment:
we got simple/table package that has minimal dependencies, maybe use it?

Reply:
yep thx